main/modemmanager/files/modemmanager

```diff
@@ -0,0 +1,6 @@
+type = process
```

Comment:
service files usually have a comment at the beginning

```suggestion
# modemmanager service

type = process
```

Reply:
tbh they're all just `# {filename} service`, not really super insightful

could probably just start getting rid of those too

Reply:
will just think about that later

Reply:
sure